utils/rpms/daos.spec

```diff
@@ -229,7 +229,7 @@ mv %{?buildroot}/%{_prefix}/etc/bash_completion.d %{?buildroot}/%{_sysconfdir}
 
 %pre server
 getent group daos_admins >/dev/null || groupadd -r daos_admins
-getent passwd daos_server >/dev/null || useradd -M daos_server
+getent passwd daos_server >/dev/null || useradd -s /sbin/nologin -r daos_server
```

Comment:
Didn't you also want to make this a system account so it doesn't have a home directory?

Reply:
Yep, that's what -r does.
```
  -r, --system                  create a system account
```